NickvisionTubeConverter.Shared/Helpers/SpeedFormatter.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+
+namespace NickvisionTubeConverter.Shared.Helpers;
+
+public static class SpeedFormatter
+{
+    private static Localizer _localizer;
+
+    static SpeedFormatter()
+    {
+        _localizer = new Localizer();
```

Comment:
`Localizer` is an `IDisposable` and it's hard to dispose this object because it is kept as a static variable and has no object lifetime.

I would not keep the localizer as a static variable here, but instead, make the `GetString` method require the localizer object:
```csharp
public static string GetString(double speed, Localizer localizer) { ... }
```